### ISSUE TITLE
Feature Request: Item Management Improvements?

### ISSUE THREAD

**Issue by Banancream12** [TangoTek/TekTopia-Community#404]
Would it be possible for villagers to leave at least 1 item in a chest and then they would go to the next stack? Also maybe if you could turn off what the villager's farm such as a cook like if you want just raw potatoes to just be able to turn off something off so the chef would stopm cooking potatoes

**Comment by orangense** [comment]
I agree that villagers should leave one item in a slot. This would be helpful in keeping my storage organized.
